legacy/web/src/main/java/org/jboss/as/web/WebConnectorDefinition.java

```diff
@@ -203,7 +204,7 @@ private WebConnectorDefinition() {
                 WebExtension.getResourceDescriptionResolver(Constants.CONNECTOR),
                 new AddressToNameAddAdaptor(CONNECTOR_ATTRIBUTES),
                 CONNECTOR_ATTRIBUTES);
-                setDeprecated(WebExtension.DEPRECATED_SINCE);
+        this.setDeprecated(WebExtension.DEPRECATED_SINCE);
```

Comment:
@bstansberry I wish but I cannot do it here since org.jboss.as.controller.ModelOnlyResourceDefinition does not expose this Parameter-based constructor variant (which resides in core).